doc/rst/platforms/cray.rst

```diff
@@ -530,6 +530,38 @@ the number of registered memory regions, so allowing a very high number of
 them may lead to reduced performance.
 
 
+ugni Hugepage-related Internal Warnings
+_______________________________________
+
+   Communication performance with ugni is so much better with hugepages
+   that if you do not use them, the runtime will print the following
+   warning when a multilocale program starts:
+
+   .. code-block:: sh
+
+      warning: without hugepages, communication performance will suffer
+
+   You can quiet this warning by giving the ``--quiet`` or ``-q`` option
+   when you run the executable.   
```

Comment:
Maybe change this to:
"""
To resolve this you can load a hugepage module with `module load craype-hugepages16M` and recompile, or if you really don't want to use hugepages you can quiet the warning by...
"""

Reply:
Will do.  (You don't have to recompile, though, just rerun.)